odl/discr/discr_ops.py

```diff
@@ -519,8 +519,8 @@ def _call(self, x, out=None):
         if out is None:
             out = self.range.element()
 
-        ndim = self.range.grid.ndim
-        dx = self.range.grid.stride
+        ndim = self.range.ndim
+        dx = self.range.cell_size
```

Comment:
I tried to avoid a plural since we have `cell_sizes` for the case it's not constant, i.e. for a `TensorGrid`. The methods in question are `cell_sizes`, `cell_boundaries` vs. `cell_size`, `cell_volume`. I admit that `cell_size` and `cell_sizes` are really close.

Maybe we could go for `cell_boundary_vecs` and `cell_size_vecs` for the first two (they return coordinate vectors) and then use `cell_sides` and `cell_volume`?